mlir/test/Dialect/SCF/parallel-loop-unroll.mlir

```diff
@@ -0,0 +1,148 @@
+// RUN: mlir-opt %s -test-parallel-loop-unrolling='unroll-factors=1,2' -split-input-file | FileCheck %s
+// RUN: mlir-opt %s -test-parallel-loop-unrolling='unroll-factors=1,2 loop-depth=1' -split-input-file | FileCheck %s --check-prefix CHECK-UNROLL-INNER
+// RUN: mlir-opt %s -test-parallel-loop-unrolling='unroll-factors=3,1' -split-input-file | FileCheck %s --check-prefix CHECK-UNROLL-BY-3
+
+func.func @unroll_simple_parallel_loop(%arg0: memref<1x16x12xf32>, %arg1: memref<1x16x12xf32>) {
```

Comment:
[nit] More descriptive names make the context much clearer. Similar comment for other tests.
```suggestion
func.func @unroll_simple_parallel_loop(%src: memref<1x16x12xf32>, %dest: memref<1x16x12xf32>) {
```